requirements.txt

```diff
@@ -0,0 +1,38 @@
+attrs==19.3.0
```

Comment:
Please only list the dependencies that are directly required, and don't mandate the versions unless it's actually necessary. Otherwise it makes it impossible to integrate in an application that uses the system python.